### DISCUSSION TITLE
Sonoff D1 dimmer >92 does not work, power ON does

### DISCUSSION THREAD

**Discussion by OldSurferDude** [arendst/Tasmota#19774] [Q&A] [answered]
On my tasmotized Sonoff D1, if I issue the command (from command line or MQTT) **power off** or **power on** everything is as exxpected.  Likewise for DIMMER nn (where nn is <93).  But numbers >92 end up turning off the light.

So I hooked it up to an oscilloscope.  (It's actually quite cool)  I execute **dimmer 10** and the delay in the start of the sine wave increases.  Then I execute dimmer 100.  The delay gets shorter and shorter and just after the point of full, unchopped sign wave, poof, no sign wave.  I'll issue the **power on** and it's on again, full sign wave.

I'll be  trying this on another Sonoff D1, but for now, something seems to be wrong translating 100 to full sign wave.  Any thoughts?  timing issue? zero crossing signal?

**Comment by sfromis** [comment, marked as answer]
You can use the command DimmerRange to reduce the hardware dimming range executed for the 0..100 values of the Dimmer command, to avoid getting outside of what's workable.

**Comment by OldSurferDude** [comment]
Thanks @sfromis That works!
